src/main/java/net/dv8tion/jda/api/requests/restaction/MessageAction.java

```diff
@@ -216,9 +216,15 @@ static boolean isDefaultMentionRepliedUser()
      * <br>This will mention the author of the target message. You can disable this through {@link #mentionRepliedUser(boolean)}.
      *
      * <p>This requires {@link net.dv8tion.jda.api.Permission#MESSAGE_HISTORY Permission.MESSAGE_HISTORY} in the channel!
+     * You cannot reply to system messages (such as {@link net.dv8tion.jda.api.entities.MessageType#CHANNEL_PINNED_ADD CHANNEL_PINNED_ADD} and similar).
```

Comment:
Are the brackets needed?
I personally think that without it will it actually sound better.

Reply:
We could remove them.

Reply:
The brackets are still here...